modules/log_ingestion.events.cft.yaml

```diff
@@ -0,0 +1,394 @@
+  TrustedIdentity:
+    Type: String
+    Description: The Role in Sysdig's AWS Account with permissions to your account
+  TargetEventBusARN:
+    Type: String
+    Description: The destination in Sysdig's AWS account where your events are sent
+  Regions:
+    Type: CommaDelimitedList
+    Description: Comma separated list of regions to monitor with EventBridge
+  OrganizationalUnitIDs:
+    Type: CommaDelimitedList
+    Description: Comma separated list of organizational unit IDs to deploy
+  RuleState:
+    Type: String
+    Description: The state of the EventBridge Rule
+    Default: ENABLED_WITH_ALL_CLOUDTRAIL_MANAGEMENT_EVENTS
+    AllowedValues:
+      - ENABLED_WITH_ALL_CLOUDTRAIL_MANAGEMENT_EVENTS
+      - ENABLED
+      - DISABLED
+  RuleEventPattern:
+    Type: String
+    Description: JSON pattern for the EventBridge rule's event pattern
+    Default: |
+      {
+        "detail-type": [
+          "AWS API Call via CloudTrail",
+          "AWS Console Sign In via CloudTrail",
+          "AWS Service Event via CloudTrail",
+          "Object Access Tier Changed",
+          "Object ACL Updated",
+          "Object Created",
+          "Object Deleted",
+          "Object Restore Completed",
+          "Object Restore Expired",
+          "Object Restore Initiated",
+          "Object Storage Class Changed",
+          "Object Tags Added",
+          "Object Tags Deleted",
+          "GuardDuty Finding"
+        ]
+      }
+  IsOrganizational:
+    Type: String
+    Description: Install into an organization
+    Default: 'false'
+    AllowedValues:
+    - 'true'
+    - 'false'
+Conditions:
+  IsOrganizational:
+    Fn::Equals:
+    - Ref: IsOrganizational
+    - 'true'
+Resources:
+  AdministrationRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub sysdig-secure-events-stackset-administration-${NameSuffix}
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: cloudformation.amazonaws.com
+          Action:
+          - sts:AssumeRole
+      Policies:
+      - PolicyName: !Sub sysdig-secure-events-stackset-execution-${NameSuffix}
+        PolicyDocument:
+          Version: 2012-10-17
+          Statement:
+          - Effect: Allow
+            Action:
+            - sts:AssumeRole
+            Resource:
+            - !Sub arn:aws:iam:::role/sysdig-secure-events-stackset-execution-${NameSuffix}
+  ExecutionRole:
+    Type: AWS::IAM::Role   
+    Properties:
+      RoleName: !Sub sysdig-secure-events-stackset-execution-${NameSuffix}
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Principal:
+            AWS:
+            - !GetAtt AdministrationRole.RoleId
+          Action:
+          - sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AdministratorAccess
```

Comment:
Isn't `AdministratorAccess` going to be more permissive?

In TF I had to add `AmazonEventBridgeFullAccess` for execution role to create stacks with EB resources. Do we not need to add it here?

Reply:
Yup, seems reasonable to pare this down.

Reply:
updated